feature_engine/datasets/data.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+import numpy as np
+
+
+def load_titanic(return_X_y_frame=False, predictors_only=False, raw=False):
+    df = pd.read_csv("https://www.openml.org/data/get_csv/16826755/phpMYEkMl")
+
+    if predictors_only:
```

Comment:
I read somewhere, that it is better to specifically say if predictors_only is True, instead of if it just exists.

Reply:
@solegalli 
Well I inspired in Sklearn's code convention. For instance this is `fit_intercept` for `LogisticRegression` and they do it this way:

https://github.com/scikit-learn/scikit-learn/blob/36958fb240fbe435673a9e3c52e769f01f36bec0/sklearn/linear_model/_logistic.py#L473

In case you think it needs to be changed,  it should go as `if predictors_only == True:` or `if predictors_only == True:`